streamalert/classifier/classifier.py

```diff
@@ -86,7 +87,9 @@ def _load_logs_for_resource(self, service, resource):
             bool: True if the resource's log sources loaded properly
         """
         # Get all logs for the configured service/entity (s3, kinesis, or sns)
-        resources = self._config['sources'].get(service)
+
+        resources = self._config['clusters'][self._cluster]['data_sources'].get(service)
+        LOGGER.debug("Resources [%s]", resources)
```

Comment:
this debug message is pretty vague.. either remove it more make it more descriptive